app/views/layouts/application.html.erb

```diff
@@ -7,8 +7,9 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
-
-<%= yield %>
+  <%= render '/layouts/navbar' %>
+  <%= yield %>
+  <%= render '/layouts/footer' %>
 
```

Comment:
because this will be on every page but the landing page we would create a different application layout for that page